spec/requests/cors_spec.rb

```diff
@@ -14,7 +14,9 @@
       expect(response.headers['Access-Control-Allow-Origin']).to eq(Rails.application.config.allowed_cors_origin)
     end
 
-    scenario 'does not allow a request from a different domain' do
+    xscenario 'does not allow a request from a different domain' do
```

Comment:
As a rule, please don't leave deactivated tests in the code.  If you need to recover it later, it's better to do it from history.